### PR TITLE
修改编译任务支持arm64版本打包

### LINKED ISSUE #42: Support ARM 64-bit architectures
**What is the feature you want to add?**

- Support ARM 64-bit architectures

**Why do you want to add this feature?**

- Support ARM 64-bit architectures

**How to implement this feature?**

- Cross-platform build

**Additional context**

When Huawei is sanctioned. We have some services run on the Huawei Kunpeng ARM64 arch platform to support Huawei. Maybe  someone else also need this.

### PR DESCRIPTION
**Pull request by mice2018**
**Please provide issue(s) of this PR: 
Fixes #42 

**To help us figure out who should review this PR, please put an X in all the areas that this PR affects.**

- [ ] Configuration
- [ ] Docs
- [x] Installation
- [ ] Performance and Scalability
- [ ] Naming
- [ ] HealthCheck
- [ ] Test and Release

**Please check any characteristics that apply to this pull request.**

- [ ] Does not have any user-facing changes. This may include API changes, behavior changes, performance improvements, etc.

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -12,11 +12,6 @@ jobs:
       matrix:
         goos: [linux, windows, darwin]
         goarch: [amd64, arm64]
-        exclude:
-          - goos: windows
```
Comment (andrewshan) [inline .github/workflows/release.yml:16]: windows的arm64不要去掉，我们只需要exclude linux的arm64即可

Comment (andrewshan) [reply .github/workflows/release.yml:16]: 还有一个test.yml也要改

**Comment by andrewshan** [comment]
单元测试跑成功了，集成测试跑失败了，原因主要是因为当前jekins只能配置一个runs-on，所以没法同一个action中跑多环境进程。

可以把集成测试部分拆离到testing-interface.yaml的文件，然后这个文件里面就不加strategy了

File: .github/workflows/testing.yml [hunk 2]
```diff
@@ -16,7 +16,29 @@ env:
 jobs:
 
   build:
-    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest, macos-latest]
+        goarch: [amd64, arm64]
+          goos: [linux, windows, darwin]
```
Comment (andrewshan) [inline .github/workflows/testing.yml:23]: 这里不应该有缩进

**Review by andrewshan — APPROVED** [review]

**Review by chuntaojun — APPROVED** [review]

**Pull request merged**